plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/CloudSdkServer.java

```diff
@@ -70,8 +70,7 @@
   public static CloudSdkServer getCloudSdkServer(IServer server) {
     CloudSdkServer cloudSdkServer = server.getAdapter(CloudSdkServer.class);
     if (cloudSdkServer == null) {
-      cloudSdkServer = (CloudSdkServer) server.loadAdapter(CloudSdkServer.class,
-                                                           new NullProgressMonitor());
+      cloudSdkServer = (CloudSdkServer) server.loadAdapter(CloudSdkServer.class, null);
```

Comment:
Null will occur is the serverType is not defined in the plugin.xml. This would cause a compile error.